ui/app/templates/components/secret-create-or-update.hbs

```diff
@@ -160,22 +164,29 @@
   >
     <div class="box is-sideless is-fullwidth is-marginless padding-top">
       <MessageError @model={{@modelForData}} @errorMessage={{this.error}} />
-      {{#if (eq @canReadSecretData false)}}
-        <AlertBanner
-          @type="warning"
-          @message="You do not have read permissions. If a secret exists here creating a new secret will overwrite it."
-          data-test-warning-no-read-permissions
-        />
-      {{/if}}
       <NamespaceReminder @mode="edit" @noun="secret" />
-      {{#if this.isCreateNewVersionFromOldVersion}}
-        <div class="form-section">
-          <AlertBanner
-            @type="warning"
-            @class="is-marginless"
-            @message="You are creating a new version based on data from Version {{@model.selectedVersion.version}}. The current version for {{@model.id}} is Version {{@model.currentVersion}}."
-          />
-        </div>
+      {{#if (or (eq @canReadSecretData false) this.isCreateNewVersionFromOldVersion)}}
+        <Hds::Alert @type="inline" @color="warning" class="has-bottom-margin-s" as |A|>
+          <A.Title>Warning</A.Title>
+          <A.Description>
+            <ul class={{if (and (eq @canReadSecretData false) this.isCreateNewVersionFromOldVersion) "bullet"}}>
+              {{#if (eq @canReadSecretData false)}}
+                <li data-test-warning-no-read-permissions>
+                  You do not have read permissions. If a secret exists here creating a new secret will overwrite it.
```

Comment:
Gosh, I hate that we have this... Something I think Chelsea and I are going to revisit. Should you write if you can't read a secret? And technically, I believe it just creates another version, you can't "update" a secret. Anyways, I digress.

Reply:
Yeah...not great. But to keep scope down for this work I tried to avoid changing any copy. I think it makes more sense to address within the KV refactor?

Reply:
Oh, 100%. And Sorry I didn't clarify. I was more or less babbling. Seeing this was a reminder for me to revisit this. I should have made that more clear. nothing for you to address here.

Reply:
Haha! I gotcha. Thanks for clarifying. Ohh the wonderful beast that is KV